src/js/midi/ui.js

```diff
@@ -0,0 +1,80 @@
+const MidiChannel = ({ type, name, id, enabled }) => {
+  const template = document.createElement('template')
+  template.innerHTML = `
+    <div style="display:flex;flex-direction:column;padding:0 10px;align-items:center">
+      <input id="${type}--${name}--${id}" style="margin:0" type="checkbox" ${
+    enabled ? 'checked="checked"' : ''
+  } />
+      <label for="${type}--${name}--${id}">${id}</label>
+    </div>
+  `
+  const content = template.content
+  content.querySelector('input[type="checkbox"]').addEventListener('change', (e) => {
+    midi.toggleChannel(type, name, id, e.target.checked)
```

Comment:
Not a fan of `toggleChannel` acting as `setChannel`. Such methods should be separate.

Reply:
Okay, I can accept that, I was stealing the idea from `.classList.toggle()`, but I'll create a proper setter.

Reply:
I see. If it's an existing idiom, I can accept it even if it's a bit weird.